far/FarRus.hlf.m4

```diff
@@ -5573,65 +5575,69 @@ $ #far:config Codepages.NoAutoDetectCP#
 $ #far:config Help.ActivateURL#
  Параметр позволяет управлять активацией URL ссылок в HLF-файлах:
 
- 0 - отключить активацию.
- 1 - активация включена.
- 2 - активация включена, но выдавать предупреждающее сообщение.
+ 0 - Активация отключена.
+ 1 - Активация включена.
+ 2 - ^<wrap>Активация включена; выдаётся предупреждающее сообщение.
 
  По умолчанию значение = 1 (разрешено).
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ Изменение этого параметра возможно через ~far:config~@FarConfig@.
 
 
 @Confirmations.EscTwiceToInterrupt
 $ #far:config Confirmations.EscTwiceToInterrupt#
- Параметр позволяет менять поведение при нажатии Esc в диалоге подтверждения прерывания операции.
-
- Может быть одним из следующих значений:
+ Параметр позволяет менять поведение клавиши #Esc# в диалоге
+подтверждения прерывания операции.
 
- 0 - ^<wrap>Нажатие кнопки ESC закрывает сообщение и продолжает выполнение операции.
- 1 - ^<wrap>Нажатие кнопки ESC закрывает сообщение и прерывает выполнение операции
+ 0 - ^<wrap>Клавиша #Esc# закрывает сообщение и продолжает выполнение операции.
+ 1 - ^<wrap>Клавиша #Esc# закрывает сообщение и прерывает выполнение операции.
 
  По умолчанию значение = 0 (закрыть сообщение и продолжить выполнение операции).
 
- Изменение этого параметра возможно через ~far:config~@FarConfig@
+ Изменение этого параметра возможно через ~far:config~@FarConfig@.
 
 
 @System.AllCtrlAltShiftRule
 $ #far:config System.AllCtrlAltShiftRule#
- Параметр задаёт поведение комбинации Ctrl+Alt+Shift для временного гашения объектов интерфейса.
+ Этот числовой параметр управляет тем, какие объекты пользовательского
+интерфейса могут быть временно скрыты комбинацией клавиш
+#Ctrl+Alt+Shift#. Каждый бит соответствует определённому типу объектов.
 
  Номера битов:
- 0 - Панели.
- 1 - Редактор.
- 2 - Внутренняя программа просмотра.
- 3 - Окно подсказки.
- 4 - Диалоги.
+ 0 – Панели;
+ 1 – Редактор;
+ 2 - Программа просмотра;
```

Comment:
Ditto, can it be consistent please?

Reply:
Argh! Replaced all en dashes with hyphens. I did not mean it. MS Word autocorrect. :( I'll be more careful in the future.